### DISCUSSION TITLE
SDK Reference?

### DISCUSSION THREAD

**Discussion by andrews05** [kreait/firebase-php#361] [Ideas]
The guides on read-the-docs are super helpful, but just wondering if there is a reference document somewhere with all of the available methods and arguments?

**Comment by jeromegamez** [comment]
Unfortunately not at the moment, until now I thought that IDE autocompletion and the GitHub code browser were enough (and I shied away from the additional build steps and deployments of the generated API docs, after finding a good generator 😅).

**Comment by andrews05** [comment]
No worries, I can get by with autocompletion :)
